lib/v2/screens/receive_screens/receive_enter_data/receive_seeds_screen.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/components/amount_entry/amount_entry_widget.dart';
+import 'package:seeds/v2/components/balance_row.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/components/text_form_field_light.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+
+class ReceiveEnterDataScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Receive", style: Theme.of(context).textTheme.headline7),
+        ),
+        body: Stack(
+          children: [
+            SingleChildScrollView(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  const SizedBox(height: 100),
+                  AmountEntryWidget(
+                    onValueChange: (value) {
+                      // BlocProvider.of<ReceiveEnterDataPageBloc>(context)
+                      //     .add(OnAmountChange(amountChanged: value));
+                    },
+                    autoFocus: true,
```

Comment:
this will make it so the widget will always autofocus, not only when you open this page the first time.

Reply:
the block will handle this part